app.js

```diff
@@ -26,8 +27,14 @@ const connector = new builder.ChatConnector({
 server.post('/api/messages', connector.listen());
 
 const bot = new builder.UniversalBot(connector, (session) => {
-    //session.send('Welcome to DotABot!');
-    session.send('Sorry, I did not understand \'%s\'. Type \'help\' if you need assistance.', session.message.text);
+
+    if(session.message.text.toLowerCase() === "help") {
```

Comment:
Have a look at [Handle user actions](https://docs.microsoft.com/en-us/bot-framework/nodejs/bot-builder-nodejs-dialog-actions) - the bot framework has some built in support for recognizing "interrupts" like this